src/content/developers/docs/transactions/index.md

```diff
@@ -305,6 +305,22 @@ Watch Austin walk you through transactions, gas, and mining.
 
 <iframe width="100%" height="315" src="https://www.youtube.com/embed/er-0ihqFQB0" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
 
+## Typed Transaction Envelope
+
+Ethereum has different transaction types that specify what a transaction should do. For example, sending Ether to an address, deploying a contract, etc.  
+Ethereum now has a new transaction standard that was defined and created by Micah Zoltu in [EIP-2718](https://eips.ethereum.org/EIPS/eip-2718): Typed Transaction Envelope, which forms the basis for some new and other yet-to-be-developed features on Ethereum.
```

Comment:
Recommend against using temporally sensitive terminology in documentation like "now has" and "that was".  The website should speak in terms of what is, largely ignoring what was unless explicitly discussing history.